pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -58,8 +58,9 @@ let
       "--with-gnu-as"
       "--with-gnu-ld"
       "--disable-debug"
-      "--enable-sjlj-exceptions"
       "--disable-win32-registry"
+    ] ++ lib.optionals (crossMingw && targetPlatform.isx86_32) [
+      "--enable-sjlj-exceptions"
```

Comment:
Can you put some/all of the commit message as a comment here? My rule of thumb is the language that describes the (new) *state* should be comments, while the language that describes the transition should be the commit message.